tests/features/user_roles_and_court.feature

```diff
@@ -0,0 +1,95 @@
+Feature: User role combined with address and court finding
+
+Tests:
+- [x] When user is defendant and address is in-state (What court is your case in?)
+- [x] When user is plaintiff and address is in-state (What court do you want to file in?)
+- [ ] When user is defendant and address is out of state (no map)
+- [ ] When user is plaintiff and address is out of state (no map)
```

Comment:
Can all four be crossed off? I think you have four tests below, and they seem to correspond to all four of these.

Reply:
You are correct! Always forget to check these off.